tests/utils/dockercli/volumelifecycle.go

```diff
@@ -30,15 +30,26 @@ import (
 // CreateVolume is going to create vsphere docker volume with given name.
 func CreateVolume(ip, name string) (string, error) {
 	log.Printf("Creating volume [%s] on VM [%s]\n", name, ip)
-	return ssh.InvokeCommand(ip, dockercli.CreateVolume+"--name="+name)
+	return ssh.InvokeCommand(ip, dockercli.CreateVolume + " --name= " + name) 
```

Comment:
nit: gofmt removes embedded spacing .. `dockercli.CreateVolume + " --name= " + name` looks cleaner and more readable but creates inconsistency with other formatting.

Supplying this comment to avoid future formatting change.